api/db-cdr/prep-tables/make-prep-survey.py

```diff
@@ -425,6 +429,8 @@ def write_row(writer, parent_id, code, name, item_type, min_value, max_value,
     if item_type == 'QUESTION':
         dict_row_register['code'] = get_short_code(code,
                                                    long_code_to_short_code)
+    if item_type != 'QUESTION':
+        dict_row_control['answers_bucketed'] = 0
```

Comment:
You can just add an else to your original if statement and your setting the wrong array. It should be dict_row_register
```suggestion
    if item_type == 'QUESTION':
        dict_row_register['code'] = get_short_code(code,
                                                   long_code_to_short_code)
   else:
        dict_row_register['answers_bucketed'] = 0
```